src/components/shared/SearchInput.js

```diff
@@ -115,7 +119,7 @@ class SearchInput extends Component {
         placeholder,
         value: query,
         spellCheck: false,
-        ref: "input"
+        ref: c => (this.input = c)
```

Comment:
Oh I could put the ref into the refs object as well by changing the line to `this.refs.input = c`